spec/support/streaming_server_manager.rb

```diff
@@ -125,6 +125,6 @@ def streaming_server_manager
   end
 
   def streaming_examples_present?
-    RUN_SYSTEM_SPECS
+    RSpec.world.filtered_examples.values.flatten.any? { |example| example.metadata[:fullstack] == true }
```

Comment:
Sort of verbose, but using same approach as the search manager class does. There may be a simpler way to check for the presence of an rspec tag...?